styles.css

```diff
@@ -71,10 +71,10 @@ details p:last-of-type {
   margin-bottom: 0;
 }
 
-/* marker cluster override */
-.marker-cluster-small div {
-  background-color: #ff8c1a !important;
-  opacity: 0.6;
+.marker-cluster, .marker-cluster div { 
+  background-color: #FF6347;
```

Comment:
use CSS color variable here (see other comment on `.color-rent-strike`)

Reply:
Great, looking forward to uDone